### PR TITLE
Issue navigation bar changes 

### PR DESCRIPTION
**Pull request by rov9**
Added wishlist page,modified wishlist page

### CODE REVIEW

File: wishlist.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+/*wishlist-page*/
+
+
+
+    *{
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+        
+    }
```
Comment (shivangisharma01) [inline wishlist.css:17]: This is used repeatedly so you should add  this in utility css file (if you've made it).

File: wishlist1.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            
+                    </td>
+                    <td><input type="number" value="1"></td>
+                    <td><button class="cart-btn">Add to cart</button></td>
+                </tr>    
+            </table>
+ 
+
+
+
```
Comment (shivangisharma01) [inline wishlist1.html:93]: remove the extra space.

**Review by shivangisharma01 — APPROVED** [review]
Made some suggestions, look into it. Apart from them it looks look.

File: login.css [hunk 3]
```diff
@@ -34,6 +34,7 @@ body{
     font-size:2rem ;
     color: #f97316;
```
Comment (tanishq20) [inline login.css:35]: you can use var() for colors.

File: wishlist.css [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+td img{
+width:80px;
+height:80px;
+margin-right: 10px;
+}
+
+.total-price{
+    display: flex;
+    justify-content: flex-end;
```
Comment (tanishq20) [inline wishlist.css:131]: It is preferred to make a utility class for all these properties.
It will be so helpful.

**Review by tanishq20 — COMMENTED** [review]
Code looks clean. Suggested minor changes.

**Pull request merged**